pkg/controller/catalogsourceconfig/catalogsourceconfig_controller.go

```diff
@@ -23,7 +24,14 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	client := mgr.GetClient()
+	// The default client serves read requests from the cache which contains
+	// objects only from the namespace the operator is watching. Given we need
+	// to query other namespaces for ConfigMaps and CatalogSources, we create
+	// create our own client and pass it the manager's scheme which has all
+	// our registered types
+	cfg, _ := config.GetConfig()
+	client, _ := client.New(cfg, client.Options{Scheme: mgr.GetScheme()})
```

Comment:
And here too.

Reply:
Done

Reply:
I think, you missed the spot here.

Reply:
The above section won't get updated. You have to go look at ec03256 commit to see the change.

Reply:
Sorry this was my bad. I indeed had missed a spot.